bindata/assets/config/defaultconfig.yaml

```diff
@@ -42,6 +42,8 @@ apiServerArguments:
     - /etc/kubernetes/static-pod-resources/configmaps/kube-apiserver-audit-policies/policy.yaml
   client-ca-file:
     - /etc/kubernetes/static-pod-certs/configmaps/client-ca/ca-bundle.crt
+  disable-admission-plugins:
```

Comment:
What about bootstrap?

Reply:
afaiu this also applies to bootstrap: https://github.com/openshift/cluster-kube-apiserver-operator/blob/cb650638006a9e6dc87c24a6faab846359a0adbb/pkg/cmd/render/render.go#L275-L276

Reply:
I see, in that case we may want to use this PR as an opportunity to remove this line:
https://github.com/openshift/cluster-kube-apiserver-operator/blob/d3025b6784e883e323f37428962e4caf4c66e3fb/bindata/bootkube/config/bootstrap-config-overrides.yaml#L60

Reply:
true, fixed :+1: